.github/workflows/e2e-test-weekly.yml

```diff
@@ -78,6 +78,13 @@ jobs:
             kubernetes-version: "v1.30"
             clusterCreation: "cli"
 
+          # Sonobuoy conformance test
+          - test: "sonobuoy conformance"
+            refStream: "ref/main/stream/debug/?"
+            attestationVariant: "gcp-sev-snp"
+            kubernetes-version: "v1.28"
```

Comment:
Can we test on v1.30 already?

Reply:
I tested v1.29. Will start another run with v1.30

Reply:
https://github.com/edgelesssys/constellation/actions/runs/10094136849